pkg/chrootuser/user.go

```diff
@@ -70,6 +70,18 @@ func GetUser(rootdir, userspec string) (uint32, uint32, error) {
 	return 0, 0, err
 }
 
+// GetGroup returns the gid by looking it up in the /etc/passwd file
+// groupspec format [ group | gid ]
+func GetGroup(rootdir, groupspec string) (uint32, error) {
+	gid64, gerr := strconv.ParseUint(groupspec, 10, 32)
+	if gerr != nil {
+		// The group couldn't be parsed as a number, so look up
+		// the group's GID.
+		gid64, gerr = lookupGroupInContainer(rootdir, groupspec)
+	}
+	return uint32(gid64), errors.Wrapf(gerr, "error looking up group for gid %q", groupspec)
```

Comment:
Is `errors.Wrapf(nil, ...)` the same thing as nil?

Even if it is, this doesn't strike me as very readable... Can you change to `if(gerr != nil) return 0, errors.Wrapf(...)`?